Documentation/ApiOverview/Bootstrapping/Index.rst

```diff
@@ -72,6 +90,10 @@ Whenever a call to TYPO3 CMS is made, the application goes through a
 bootstrapping process managed by a dedicated API. This process is also
 used in the frontend, but only the backend process is described here.
 
+
+Classes involved in the backend bootstrapping process are :php:`\TYPO3\CMS\Core\Core\Bootstrap`,
```

Comment:
There are *many* more classes involved so what was the intention of mentioning these?

Reply:
Removed the sentence